src/test/java/com/antkorwin/xsyncexamples/Iteration.java

```diff
@@ -0,0 +1,17 @@
+package com.antkorwin.xsyncexamples;
+
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+/**
+ * @author Arthur Kupriyanov on 08.02.2020
+ */
+public class Iteration {
+    public static IntStream getIterations(int range){
+        return IntStream.range(0 ,range);
+    }
+
+    public static Stream<Integer> getParallelIterations(int range){
+        return getIterations(range).boxed().parallel();
```

Comment:
I have written the stress testing tools to run test code concurrently with an ability to set an execution mode. 
https://github.com/jupiter-tools/stress-test#rest-api-stress-testing

It provides more strong guarantees for the number of executed threads than the parallel streams in java. Sometimes java parallel streams may lead to flaky tests, when you expecting a true concurrently execution of multiple threads but the stream runs in a single thread.

You can find an example of usage of this library in XSync tests:
https://github.com/antkorwin/xsync/blob/5a1bb6c682db0239291996a4bbe31c921359e14a/src/test/java/com/antkorwin/xsync/XSyncTest.java#L155

It will be good to replace all uses of parallel streams in tests on the StressTestRunner.